lib/util/fs/util.ts

```diff
@@ -0,0 +1,21 @@
+import upath from 'upath';
+import { GlobalConfig } from '../../config/global';
+import { assertBaseDir } from './access';
+
+export function ensureLocalPath(path: string): string {
```

Comment:
Are you sure that the path will always be resolved before being passed to this function?

e.g. let's say that the localDir is `/tmp/renovate`. What if `path` is `/tmp/renovate/../../etc/passwd`?

Reply:
Fixed and updated tests accordingly